src/sync/createInternalRecord.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+/**
+ * Methods to create internal records from a requested
+ * sync/external record. Used primarily by
+ * createOrUpdateRecord in incomingSyncUtils.
+ */
+
+import { parseBoolean } from './incomingSyncUtils';
+
+export const createPeriodInternalRecord = (record, database) => ({
+  id: record.ID,
+  startDate: record.startDate,
+  endDate: record.endDate,
+  name: record.name,
+  periodSchedule: database.getOrCreate('PeriodSchedule', record.periodScheduleID),
+});
+
+export const createOptionsInternalRecord = record => ({
+  id: record.ID,
+  title: record.title,
+  type: record.type,
+  isActive: parseBoolean(record.isActive),
+});
+
+export const createPeriodScheduleInternalRecord = record => ({
+  id: record.ID,
+  name: record.name,
+});
```

Comment:
2 criticisms:
- If this is the pattern we follow and refactored the rest of incomingSyncUtils.js we'd just be moving the problem (file annoyingly big)
- This could be confused with `createRecord.js`?